overlay_all_stats.py

```diff
@@ -32,86 +32,112 @@ def _update_data_layer(self):
         self.data_canvas.clear()
         self.message_canvas.clear()
 
-        if self.data["power"] != 0:
+        if self.data["power"].get() != 0:
             self.draw_power_rec_power()
 
-        if self.data["cadence"] != 0:
+        if self.data["cadence"].get() != 0:
             self.draw_cadence()
 
-        if self.data["reed_velocity"] != 0:
+        if self.data["reed_velocity"].get() != 0:
             self.draw_max_rec_reed_velocity()
 
-        if self.data["reed_distance"] != 0:
+        if self.data["reed_distance"].get() != 0:
             self.draw_distance()
 
         if self.data.has_message():
             self.draw_messages()
 
     def draw_power_rec_power(self):
-        power = self.data["power"]
-        rec_power = self.data["rec_power"]
+        power = self.data["power"].get()
+        rec_power = self.data["rec_power"].get()
```

Comment:
It would be "safer" to only call `.get()` below where we've called `.is_valid()`, as this way we never end up with potentially invalid variables lying around which may be mistakenly used.

This applies to a few places throughout both overlays.